tidy3d/components/grid/grid_spec.py

```diff
@@ -277,6 +277,14 @@ class AutoGrid(GridSpec1d):
         lt=2.0,
     )
 
+    global_min_dl: pd.NonNegativeFloat = pd.Field(
+        0,
+        title="Lower bound of grid size",
+        description="The lower bound of grid size along this dimension in the entire "
+        "computational domain. It's a soft bound, meaning that the actual minimal "
+        "grid size might be slightly smaller.",
+    )
```

Comment:
Rewording a little - also am I understanding it correctly?

```
Lower bound of the grid size along this dimension regardless of structures present in the simulation, including override structures with `enforced=True`. It is a soft bound, meaning that the actual minimal grid size might be slightly smaller.
```

Reply:
Good rewording!